snowpack/src/commands/dev.ts

```diff
@@ -803,7 +820,8 @@ ${err}`);
     })
     .listen(port);
 
-  const hmrEngine = new EsmHmrEngine({server});
+  const liveReloadDelayMs = config.devOptions.liveReloadDelayMs;
```

Comment:
bikeshed: I know the original request was for `liveReloadDelayMs`, but I'm realizing now that that name isn't really correct / is too specific, since HMR and live-reload are technically two different things (and live-reload is driven through our HMR engine, as you can see here.

Can we:
1. change the Snowpack config value to `devOptions.hmrDelay`
2. change the `EsmHmrEngine` option name to `delay`

So that this would become:

```
const {hmrDelay} = config.devOptions;
const hmrEngine = new EsmHmrEngine({server, delay: hmrDelay});
```